libs/drupal_acces.js

```diff
@@ -0,0 +1,136 @@
+var request = require('superagent');
+var logger = require('../libs/logger')(module);
+
+var DRUPAL_PROFILE_API_URL = process.env.DRUPAL_PROFILE_API_URL;
+var DRUPAL_PROFILE_API_USER = process.env.DRUPAL_PROFILE_API_USER;
+var DRUPAL_PROFILE_API_PASSWORD = process.env.DRUPAL_PROFILE_API_PASSWORD;
+
+function _parse_fron_anchor_tag(uid_string) {
+    var re = /<a.*>(.+)<\/a>/g;
+    m = re.exec(uid_string);
+    re.lastIndex = 0;
+    if (m.length === 2) {
+        return m[1];
+    } else {
+        return undefined;
+    }
+}
+
+function _parse_user(res_body) {
+    if (res_body.length === 0) {
+        return undefined
+    }
+    has_ticket_ = res_body[0]['PHP'] !== "";
```

Comment:
what is the need to call a variable with an underscore at the end? 
where is `has_ticket_` defined?